### PR TITLE
Sentry kubernetes

### PR DESCRIPTION
**Pull request by gianrubio**
This PR is a review from the closed one #2708, I did all the changes requested by @unguiculus.

### CODE REVIEW

**Review by unguiculus — CHANGES_REQUESTED** [review]
Our RBAC best practices have been revised. Please update accordingly.

https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md

**Comment by gianrubio** [comment]
@unguiculus done

File: incubator/sentry-kubernetes/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for sentry-kubernetes (https://github.com/getsentry/sentry-kubernetes)
+name: sentry-kubernetes
+version: 0.1.0
```
Comment (unguiculus) [inline incubator/sentry-kubernetes/Chart.yaml:4]: Add `appVersion` and `icon`, `sources`, `keywords`, etc. if possible.

File: incubator/sentry-kubernetes/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+*/}}
+{{- define "sentry-kubernetes.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "sentry-kubernetes.fullname" -}}
```
Comment (unguiculus) [inline incubator/sentry-kubernetes/templates/_helpers.tpl]: Templates have been revised. Please have a look at the `_helpers.tpl` you get with `helm create` using Helm 2.8+. Please update.

File: incubator/sentry-kubernetes/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
```
Comment (unguiculus) [inline incubator/sentry-kubernetes/templates/deployment.yaml]: `apps/v1beta2`

File: incubator/sentry-kubernetes/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+kind: Deployment
+metadata:
```
Comment (unguiculus) [inline incubator/sentry-kubernetes/templates/deployment.yaml]: Indent two spaces.

File: incubator/sentry-kubernetes/templates/secret.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
```
Comment (unguiculus) [inline incubator/sentry-kubernetes/templates/secret.yaml:3]: Indent two spaces.

File: incubator/sentry-kubernetes/templates/serviceaccount.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
```
Comment (unguiculus) [inline incubator/sentry-kubernetes/templates/serviceaccount.yaml:4]: Indent two spaces.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please use a consistent list indentation style. I'd suggest you always indent two spaces. Currently, it's a mixture of zero and two spaces.

**Comment by gianrubio** [comment]
@unguiculus your requested changes are done

File: incubator/sentry-kubernetes/templates/deployment.yaml [hunk 7]
```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta2
```
Comment (unguiculus) [inline incubator/sentry-kubernetes/templates/deployment.yaml]: `apps/v1beta2`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/sentry-kubernetes/templates/deployment.yaml (hunk 7 above)
Comment (gianrubio) [reply incubator/sentry-kubernetes/templates/deployment.yaml]: sorry, fixed

**Review by unguiculus — APPROVED** [review]

**Pull request merged**